### ISSUE TITLE
[accessibility] textarea char counter warning should work for color-blind people

### ISSUE THREAD

**Issue by smknstd** [code16/sharp#504]
![Screenshot 2023-10-19 at 10-57-23 New “entity” Posts Demo project (Sharp 8 1 2)](https://github.com/code16/sharp/assets/2412608/3741bf12-3398-47b7-ad31-d1e250df2b87)

Maybe it should have an icon :

![unnamed(6)](https://github.com/code16/sharp/assets/2412608/689de6a7-89c0-4812-a35a-16eaa6a4e559)

Bonus: exceeded chars could be highlighted

![unnamed(8)](https://github.com/code16/sharp/assets/2412608/f8a8a8e7-5991-4f9d-ba81-df1c84367c18)
